APE24.rst

```diff
@@ -0,0 +1,443 @@
+* `Metrics for community partnership landing pages to track health <https://github.com/pyOpenSci/software-peer-review/issues/226>`_
+  (as noted above in **"Re-review" of packages**, this issue is not required
+  to be resolved for this APE to be accepted)
+
+Pull requests:
+
+* `Feat: add text associated with astropy partnership <https://github.com/pyOpenSci/software-peer-review/pull/225>`_
+* `FEAT: Start at community mockup / astropy <https://github.com/pyOpenSci/pyopensci.github.io/pull/207>`_
+* *TODO: A future PR by William Jamieson to update our Affiliated guidelines and process,
+  after consolidating what we have with that of pyOpenSci.*
+
+
+Implementation
+==============
+
+.. This section lists the major steps required to implement the APE.  Where
+.. possible, it should be noted where one step is dependent on another, and which
+.. steps may be optionally omitted.  Where it makes sense, each  step should
+.. include a link related pull requests as the implementation progresses.
+
+If we decide to move forward with this partnership, these are the proposed steps:
+
+1. Do a trial review period of some packages to see how it goes to make the
+   Astropy community feel more comfortable with the changes
+   (also see **The trial period** below).
+2. Based on the trial, our current Affiliated editors would finalize
+   and publish the updated guidelines and process, including new
+   COI policy, packaging guidelines, and so forth.
+   These new guidelines need to be compatible with
+   `pre-APE 24 Affiliated Package review guidelines`_ and
+   `pre-APE 24 Affiliated Package review process, with templates`_.
+   At the same time, we should link to more modern
+   practices over at pyOpenSci and `Scientific Python`_, in addition
+   to the "old ways" in Astropy and OpenAstronomy.
+3. Come up with a plan to transition already accepted packages over to
+   the pyOpenSci review process. This could be a process that happens over time
+   in that new packages just go through the new review process and get the value
+   of this partnership (pyOpenSci and JOSS) through that transition.
+   Then, we could slowly look at the older packages and evaluate their current
+   health states to determine whether another review is warranted
+   (also see **"Re-review" of packages**). We will encourage these older
+   packages that are still actively maintained to also go through the
+   new process, pointing them to the benefits mentioned in
+   **What do we gain from a partnership with pyOpenSci?**
+4. For the new packages, have our project website ingest a RSS/XML feed
+   from pyOpenSci for cross-listing (also see **Listing on websites**).
+   We would also link back to pyOpenSci on our page.
+   Meanwhile, we would keep the pre-APE 24 listing on a "legacy affiliation" page
+   that will be kept alive during the lifetime of the Astropy project.
+5. Work with pyOpenSci to cross-list Affiliated editors, between their
```

Comment:
this is just an idea that i just had - but i was thining it might be cool to have a little astropy logo on the [editor listing page](https://www.pyopensci.org/about-peer-review/index.html#our-editorial-board) that would highlight astropy editors. it would also provide more visibility for astropy on multiple pages on our website. :)

Reply:
Added your idea to the APE. Thanks!